filebeat/harvester/log.go

```diff
@@ -44,6 +44,29 @@ func (h *Harvester) Harvest() {
 		return
 	}
 
+	go func() {
+		select {
+		case <-h.prospectorDone:
+			close(h.done)
+		case <-h.done:
+			return
+		}
+
+	}()
+
+	if h.config.CloseTimeout > 0 {
+		go func() {
+			select {
+			case <-h.done:
+				return
+			case <-time.After(h.config.CloseTimeout):
+				close(h.done)
```

Comment:
If I remember correctly, h.done is passed to line reader. So this should break the reader loop. But I'm not sure about impact on multiline. Will a multiline event being generated if h.done is closed?